### ISSUE TITLE
Testing Code and Pre-Trained Model

### ISSUE THREAD

**Issue by afung-git** [grey-eye/talking-heads#31]
Hi @MrCaracara,

This is an awesome implementation for the paper! Was wondering if you would eventually upload your pre-trained model for the meta-learning as well as a testing script so we can play around with it.

**Comment by castelo-software** [comment]
Hi @afung-git,

I could upload the weights, but you still can't do any fine-tuning with them, as I haven't trained with Loss_MCH, which is necessary to ensure similarity between the W matrix columns and the outputs of the embedder.
I am quite busy right now to fix the problem with this loss, but if you want to give fine-tuining a try, feel free to try and fix it!

**Comment by usingcolor** [comment]
@MrCaracara Can you give me a weights of the meta trained networks? If I can use them, it will be very helpful for my work.

**Comment by mohsin127** [comment]
Hey If you can also provide me with the pretrained models that would be great 
Thanks!
